package.json

```diff
@@ -120,5 +120,8 @@
       "packages/*",
       "packages/react-integration/demo-app-ts"
     ]
+  },
+  "dependencies": {
```

Comment:
I'd wager that this shouldn't be a necessary change? What happens if you remove this line and run things locally (`yarn install && yarn start`)?